// Try & Catch

/* The try statement allows you to define a block of code 
    to be tested for errors while it is being executd.*/

/* The catch statement allows you to define a block of code 
    to be executed, if an error occurs in try block */

    try{
        console.log(a);
    } catch (e){
        console.log(e);
    }
      
     // o/p => a is not defined...